jdaviz/app.py

```diff
@@ -105,6 +108,9 @@ class ApplicationState(State):
     tray_items = ListCallbackProperty(
         docstring="List of plugins displayed in the sidebar tray area.")
 
+    tray_open = CallbackProperty(
+        [], docstring="The plugin(s) opened in sidebar tray area.")
+
```

Comment:
Should we rename the `drawer` CallbackProperty (and reorder it with the others so it's not isolated at the top)?  On first read I couldn't tell the difference until I looked to see where/how they were both used, so perhaps `drawer` -> `tray_open` and `tray_open` -> `tray_items_open`?

Reply:
Yes, `tray_items_open` is a better name. The renaming of drawer should not go into this PR IMO.

Reply:
Can we rename stuff without breaking stuff? Some of this `Application` setup is pretty fragile.